code/SS_Datetimezone.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Class SS_Datetimezone
+ *
+ * Adds customisable timezones to the nice method on {@link SS_Datetime}.
+ */
+class SS_Datetimezone extends SS_Datetime
+{
+    /**
+    * Returns the date in the raw SQL-format specific to a given timezone passed from the Member class, e.g. “2006-01-18 16:32:04”
+    */
+    public function Nice() {
+        if($timestamp = $this){
```

Comment:
this is not really good practise, as changing anything that is `$this` (even redefined) will update it by reference, will it not?

Might be a better idea to do something like this:

```$timezone = clone $this;```